Source/WebCore/Modules/paymentrequest/MerchantValidationEvent.cpp

```diff
@@ -43,7 +43,7 @@ Ref<MerchantValidationEvent> MerchantValidationEvent::create(const AtomString& t
 
 ExceptionOr<Ref<MerchantValidationEvent>> MerchantValidationEvent::create(Document& document, const AtomString& type, Init&& eventInit)
 {
-    URL validationURL { document.url(), eventInit.validationURL };
+    URL validationURL { document.completeURL(eventInit.validationURL) };
```

Comment:
I think this style ends up calling the URL constructor redundantly, so I'd suggest:
```suggestion
    auto validationURL = document.completeURL(eventInit.validationURL);
```

Reply:
Done.